apps/server/src/utils/parserFunctions.ts

```diff
@@ -48,11 +48,17 @@ export const parseRundown = (data: Partial<DatabaseModel>): OntimeRundown => {
     let newEvent: OntimeEvent | OntimeDelay | OntimeBlock | null;
 
     if (isOntimeEvent(event)) {
+      const tryToLink = event.linkStart;
       newEvent = createEvent(event, eventIndex.toString());
       // skip if event is invalid
       if (newEvent == null) {
         continue;
       }
+      const prevEvent = rundown.at(-1);
+      if (tryToLink && isOntimeEvent(prevEvent)) {
+        newEvent.linkStart = prevEvent.id;
+        newEvent.timeStart = prevEvent.timeEnd;
```

Comment:
We should rethink this code.

The previous might not be an event, in which case we don't have a value 

The values here will be calculated by `rundownCache`

I think we either 
a) always provide a value
b) never provide a value 
c) ignore the value

The point being that the output here has to be predicable